homeassistant/components/vizio/media_player.py

```diff
@@ -127,100 +144,125 @@ def update(self):
             self._available_inputs = None
 
     @property
-    def state(self):
+    def state(self) -> str:
         """Return the state of the device."""
+
         return self._state
 
     @property
-    def name(self):
+    def name(self) -> str:
         """Return the name of the device."""
+
         return self._name
 
     @property
-    def icon(self):
+    def icon(self) -> str:
         """Return the icon of the device."""
+
         return self._icon
 
     @property
-    def volume_level(self):
+    def volume_level(self) -> float:
         """Return the volume level of the device."""
+
         return self._volume_level
 
     @property
-    def source(self):
+    def source(self) -> str:
         """Return current input of the device."""
+
         return self._current_input
 
     @property
-    def source_list(self):
+    def source_list(self) -> List:
         """Return list of available inputs of the device."""
+
         return self._available_inputs
 
     @property
-    def supported_features(self):
+    def supported_features(self) -> int:
         """Flag device features that are supported."""
+
         return self._supported_commands
 
     @property
-    def unique_id(self):
+    def unique_id(self) -> str:
         """Return the unique id of the device."""
+
         return self._unique_id
 
-    def turn_on(self):
+    @property
+    def device_info(self) -> Dict[str, Any]:
```

Comment:
Wait, sorry, this won't have an effect at this point. Only config entry integrations can use `device_info`. Please remove this.